app/assets/stylesheets/base.scss

```diff
@@ -0,0 +1,47 @@
+main {
+
+  h1 {
+    @extend .heading-xlarge !optional;
```

Comment:
In the `Gemfile.lock` we are using sass 3.4.22 and it is the exact error message.